File: TransCoder_tokenized_test_set_functions/java/SMALLEST_NUMBER_SUM_DIGITS_N_DIVISIBLE_10N.java
static void digitsNum ( int N ) {
  if ( N == 0 ) System . out . println ( "0" );
  if ( N % 9 != 0 ) System . out . print ( ( N % 9 ) );
  for ( int i = 1;
  i <= ( N / 9 );
  ++ i ) System . out . print ( "9" );
  for ( int i = 1;
  i <= N;
  ++ i ) System . out . print ( "0" );
  System . out . print ( "" );
}

File: TransCoder_tokenized_test_set_functions/python/SMALLEST_NUMBER_SUM_DIGITS_N_DIVISIBLE_10N.py
def digitsNum ( N ) :
    if ( N == 0 ) :
        print ( "0", end = "" )
    if ( N % 9 != 0 ) :
        print ( N % 9, end = "" )
    for i in range ( 1, int ( N / 9 ) + 1 ) :
        print ( "9", end = "" )
    for i in range ( 1, N + 1 ) :
        print ( "0", end = "" )
    print ( )
